### PR TITLE
feat: New table data apis endpoints

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by dstala — CHANGES_REQUESTED** [review]
- [ ] Links list currently doesn't work as desired for BT
- [ ] Links list for HM returns an extra field
Screenshot 2023-06-16 at 11 00 30 AM

File: packages/noco-docs/content/en/developer-resources/rest-apis.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ Currently, the default value for {orgs} is <b>noco</b>. Users will be able to ch
 | Data | Delete| dbViewRow | delete | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/{rowId} |
 | Data | Get | dbViewRow | count | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/count |
 | Data | Get | dbViewRow | groupedDataList | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/group/{columnId} |
+| Data | Get | dbTableRow | tableRowList | /api/v1/tables/{tableId} |
+| Data | Post | dbTableRow | tableRowCreate | /api/v1/tables/{tableId} |
+| Data | Get | dbTableRow | tableRowRead | /api/v1/tables/{tableId}/rows/{rowId} |
+| Data | Patch | dbTableRow | tableRowUpdate | /api/v1/tables/{tableId}/rows |
+| Data | Delete| dbTableRow | tableRowDelete | /api/v1/tables/{tableId}/rows |
```
Comment (wingkwong) [inline packages/noco-docs/content/en/developer-resources/rest-apis.md:81]: `{rowId}` is missing.

File: packages/noco-docs/content/en/developer-resources/rest-apis.md [hunk 2]
```diff
@@ -74,6 +74,12 @@ Currently, the default value for {orgs} is <b>noco</b>. Users will be able to ch
 | Data | Delete| dbViewRow | delete | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/{rowId} |
 | Data | Get | dbViewRow | count | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/count |
 | Data | Get | dbViewRow | groupedDataList | /api/v1/db/data/{orgs}/{projectName}/{tableName}/views/{viewName}/group/{columnId} |
+| Data | Get | dbTableRow | tableRowList | /api/v1/tables/{tableId} |
+| Data | Post | dbTableRow | tableRowCreate | /api/v1/tables/{tableId} |
+| Data | Get | dbTableRow | tableRowRead | /api/v1/tables/{tableId}/rows/{rowId} |
```
Comment (wingkwong) [inline packages/noco-docs/content/en/developer-resources/rest-apis.md:79]: not found this API in Api.ts

**Review by dstala — APPROVED** [review]

File: packages/noco-docs/content/en/developer-resources/rest-apis.md (hunk 1 above)
Comment (pranavxc) [reply packages/noco-docs/content/en/developer-resources/rest-apis.md:81]: `{rowId}` is part of the body

File: packages/noco-docs/content/en/developer-resources/rest-apis.md (hunk 2 above)
Comment (pranavxc) [reply packages/noco-docs/content/en/developer-resources/rest-apis.md:79]: We will remove these from docs

**Pull request merged**